lib/smart_proxy_remote_execution_ssh/command_update.rb

```diff
@@ -0,0 +1,73 @@
+module Proxy::RemoteExecution::Ssh
+  # update sent back to the suspended action
+  class CommandUpdate
+    attr_reader :buffer, :exit_status
+
+    def initialize(buffer)
+      @buffer      = buffer
```

Comment:
White-space commit pushed (I've also updated some alignations on =>?, as I know you don't like it:)